test/bigtest/network/factories/order.js

```diff
@@ -0,0 +1,14 @@
+import { Factory } from 'miragejs';
+import faker from 'faker';
+
+import {
+  ORDER_STATUSES,
+} from '@folio/stripes-acq-components';
+
+export default Factory.extend({
```

Comment:
I don't think it's the best idea to copy-paste order models and factories round repos, maybe it makes sense to move them under stripes-acq-components

Reply:
moved config, model, serializer for orders to `stripes-acq-components` - [PR](https://github.com/folio-org/stripes-acq-components/pull/220)